packages/neuron-ui/src/components/WalletEditor/index.tsx

```diff
@@ -79,36 +68,46 @@ export default (props: React.PropsWithoutRef<ContentProps & RouteComponentProps<
             <InlineInput {...inputProps} key={inputProps.label} />
           ))}
         </Form>
-        <Button
-          type="submit"
-          variant="primary"
-          size="lg"
-          block
-          onClick={() => handleSubmit()}
-          disabled={password === '' || confirmPassword === '' || password !== confirmPassword || walletName === ''}
-        >
+        <Button type="submit" variant="primary" size="lg" block onClick={handleSubmit} disabled={!isParamsValid}>
           {t('common.save')}
         </Button>
       </Card.Body>
       <Dialog
         open={dialog.open}
         onClick={() => {
-          dispatch({
-            type: MainActions.SetDialog,
-            payload: {
-              open: false,
-            },
-          })
+          toggleDialog(false)
         }}
       >
-        <InputWalletPasswordDialog
-          wallet={myWallet}
-          dispatch={dispatch}
-          checkType={CheckType.EditWallet}
-          errorMessage={errorMessages.length > 0 ? errorMessages[errorMessages.length - 1].content : ''}
-          newWalletName={walletName}
-          newPassword={password}
-        />
+        <Card
+          onClick={(e: React.SyntheticEvent<HTMLDivElement>) => {
+            e.preventDefault()
+            e.stopPropagation()
+          }}
+          style={{
+            width: '40%',
+          }}
+        >
+          <>
+            <Card.Header>
+              {t('settings.wallet-manager.delete-wallet-title', { name: wallet ? wallet.name : '' })}
```

Comment:
Under what circumstances would a passed wallet be `nil`? Is it still making any sense to show the dialog when the wallet is not found?

Reply:
it could be removed caused `wallet` check has been executed at https://github.com/nervosnetwork/neuron/pull/259/files/6bc70c4452aa5ae33e8a0147592a2d4a7a340926#diff-764ef9b4eee7844302497a22e71cadadR26

Reply:
updated in https://github.com/nervosnetwork/neuron/pull/259/commits/19789de6f0dcab7327d7c86981bf8f5fa97127e9